### ISSUE TITLE
Error with Demuxmix

### ISSUE THREAD

**Issue by mari-ga** [huklein/demuxmix#12]
Hi! 

First of all, thank you very much for this tool. 
I wanted to report an error which I've constantly found when trying different datasets, using the naive model and other models including RNA data.

```
> demuxmix_demul <- demuxmix(hto_counts,model="naive")

Error in while (iter < maxIter & abs(deltaLogLik/logLik) > tol) { : 
  missing value where TRUE/FALSE needed
In addition: Warning message:
In dnbinom(hto, mu = mu2, size = theta2) : NaNs produced
```

Any idea of why is causing this error?

Thank you very much in advance.

**Comment by huklein** [comment]
Hi!
Thank you for reporting this. I haven't observed this specific error myself and only have a broad idea of what may be causing it. If you don't mind, could you please send me the HTO data either as an R file or CSV file to hansulrich.klein@gmail.com? I am very interested in solving this issue, especially since you mentioned observing it in more than one dataset.

**Comment by huklein** [comment]
Has been addressed in version 1.5.1. A warning is issued instead of an error and the model is returned for debugging purposes.
